apps/src/util/CourseBlockCardsConstants.js

```diff
@@ -0,0 +1,130 @@
+import i18n from '@cdo/locale';
```

Comment:
nit: it seems like normally our non-react files begin with a lowercase letter. Would it make sense to follow that convention here? (slash is that actually a convention?)

Reply:
Good catch - changed!

Reply:
idea: a plugin like this one to enforce filename convention: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/filename-case.md